src/test/java/be/seeseemelk/mockbukkit/inventory/meta/BookMetaMockTest.java

```diff
@@ -0,0 +1,83 @@
+package be.seeseemelk.mockbukkit.inventory.meta;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+
+public class BookMetaMockTest
+{
+	private BookMetaMock meta;
+
+	@BeforeEach
+	void setUp()
+	{
+		MockBukkit.mock();
+		meta = new BookMetaMock();
+	}
+
+	@AfterEach
+	void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
+	@Test
+	void constructor_DefaultValues()
+	{
+		assertNull(meta.getTitle());
+		assertNull(meta.getAuthor());
+		assertNull(meta.getGeneration());
+		assertEquals(meta.getPageCount(), 0);
+
+		assertFalse(meta.hasAuthor());
+		assertFalse(meta.hasTitle());
+		assertFalse(meta.hasGeneration());
+		assertFalse(meta.hasPages());
+	}
+
+	@Test
+	void constructor_Clone_CopiesValues()
```

Comment:
There's two ways to clone BookMetaMock, one is through the constructor (which is covered here) and another is in the Object#clone() method. Although it looks like that will work here anyhow, I think it's always a good idea to add that test

Another thing (which is an general issue not thought that much of in Mockbukkit to be real here) is that Paper almost always deep clones the objects; all mutable fields of the object has to be cloned as well (recursively). In this case it's fine implementation wise, but feel free to check if the mutable field `pages` changes reference but keeps the same contents between the two copies.

Reply:
Ok, done.